cmd/argo/commands/clustertemplate/create.go

```diff
@@ -25,6 +25,16 @@ func NewCreateCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "create FILE1 FILE2...",
 		Short: "create a cluster workflow template",
+        Example: `# Create a Cluster Workflow Template:
```

Comment:
this is supposed to be 2 tabs instead of spaces. GH doesn't allow tabs in suggestions though, so that may be due to my previous suggestion containing spaces, sorry about that 😕